packages/react-vapor/tsconfig.json

```diff
@@ -2,13 +2,13 @@
     "extends": "../tsconfig-base",
     "compilerOptions": {
         "outDir": "dist",
-        "rootDir": "src",
         "tsBuildInfoFile": "./dist/.tsbuildinfo",
         "baseUrl": ".",
         "paths": {
-            "@test-utils": ["src/TestUtils.tsx"],
+            "@test-utils": ["jest/utils.tsx"],
             "*": ["types/*", "*"]
         }
     },
-    "include": ["src"]
+    "include": ["src", "jest"],
+    "exclude": ["**/*.js"]

```

Comment:
This won't exclude the node_modules, dist and coverage folder 🤔

Reply:
[`exclude` only exclude stuff that is included by `include`](https://www.typescriptlang.org/tsconfig#exclude). `node_modules`, `dist` and `coverage` folders aren't under either `src` or `jest`. So we are good here. You can confirm by running `tsc --showConfig` to see what files are included in compilation.